packages/terra-core-docs/src/terra-dev-site/doc/hyperlink/UpgradeGuide.2.doc.mdx

```diff
@@ -4,6 +4,22 @@ import { Badge } from 'terra-hyperlink/package.json?dev-site-package';
 
 # Terra Hyperlink Upgrade Guide
 
+## Changes from version 2 to version 3
+
+### Changes to Props
+
+* `isDisabled` prop has been marked as **private**, allowing it to be used, but for very special circumstances or higher-order components.
```

Comment:
I'm curious about this pattern. Do we intend for our consumers to use this prop when they make their own higher order components, or do we intend for only the Terra team to use this prop when we are creating additional Terra components?

Reply:
We had a discussion with neil and scott whether or not to remove these props completely and then decided that there might be few consumers who are using hyperlink as an HOC like a menu item or something else where they might be using these props so decided to go with private only.